app/assets/javascripts/admin/addon/components/theme-settings-editor.js

```diff
@@ -0,0 +1,190 @@
+import Component from "@ember/component";
+import { action, computed } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+import { ajax } from "discourse/lib/ajax";
+import { inject as service } from "@ember/service";
+import I18n from "I18n";
+
+export default class ThemeSettingsEditor extends Component {
+  @service dialog;
+
+  @tracked editedContent = null;
```

Comment:
tracked values are `undefined` by default, so you should be able to just define the property with no default value.

```suggestion
  @tracked editedContent;
```